src/main/java/frc/robot/Robot.java

```diff
@@ -670,7 +670,7 @@ private void executeClimbingStateMachine() {
         mClimber.extend();
 
         /** Checks if the motor is overcurrenting, signalling it's done climbing */
-        if (mClimber.checkOvercurrent()) {
+        if (mClimber.readyForNextState()) {
```

Comment:
This should be: `if (mClimber.isExtended())`

Reply:
Name changed